io/net/bridge.py

```diff
@@ -82,20 +83,19 @@ def test_bridge_run(self):
                                             if_type="Bridge")
         try:
             networkinterface.add_ipaddr(self.ipaddr, self.netmask)
-            networkinterface.save(self.ipaddr, self.netmask)
         except Exception:
             networkinterface.save(self.ipaddr, self.netmask)
         networkinterface.bring_up()
         if networkinterface.ping_check(self.peer_ip, count=5) is not None:
             self.fail('Ping using bridge failed')
         networkinterface.remove_ipaddr(self.ipaddr, self.netmask)
+        try:
+            networkinterface.restore_from_backup()
+        except Exception:
+            self.log.info("backup file not availbale, could not restore file.")
 
     def test_bridge_delete(self):
         '''
         Set to original state
         '''
         self.check_failure('ip link del dev %s' % self.bridge_interface)
-        # TODO:need to get this functionality into avocado utils interfcae.py
-        path = "/etc/sysconfig/network-scripts/ifcfg-%s" \
```

Comment:
why this things removed,we have added this when cfg file will present, bridge interface is coming after delete also.

Reply:
yes,for sles we need to add here as restore_from_backup() will not remove cfg file.

Reply:
@vaishnavibhat can you take care of this,don't remove these lines.

Reply:
@smruti77 these lines are not there in the original file. I have included in the patch only.

Reply:
it was there in original file,may be your master old one.